[0.034s] Invoking command in '/home/nicholas/arduinobot_ws/build/arduinobot_msgs': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/nicholas/arduinobot_ws/install/arduinobot_py_examples:/opt/ros/humble /usr/bin/cmake --build /home/nicholas/arduinobot_ws/build/arduinobot_msgs -- -j16 -l16
[0.069s] -- Found ament_cmake: 1.3.11 (/opt/ros/humble/share/ament_cmake/cmake)
[0.190s] -- Found std_msgs: 4.8.0 (/opt/ros/humble/share/std_msgs/cmake)
[0.214s] -- Found rosidl_generator_c: 3.1.6 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.216s] -- Found rosidl_adapter: 3.1.6 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.222s] -- Found rosidl_generator_cpp: 3.1.6 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.232s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.243s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.262s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.463s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.463s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.463s]   arduinobot_msgs --arguments-file
[0.463s]   /home/nicholas/arduinobot_ws/build/arduinobot_msgs/rosidl_adapter__arguments__arduinobot_msgs.json
[0.463s]   --output-dir
[0.463s]   /home/nicholas/arduinobot_ws/build/arduinobot_msgs/rosidl_adapter/arduinobot_msgs
[0.463s]   --output-file
[0.463s]   /home/nicholas/arduinobot_ws/build/arduinobot_msgs/rosidl_adapter/arduinobot_msgs.idls)
[0.463s]   returned error code 1:
[0.463s] 
[0.463s]   Traceback (most recent call last):
[0.463s] 
[0.463s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.463s]       return _run_code(code, main_globals, None,
[0.463s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.463s]       exec(code, run_globals)
[0.463s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.464s]       sys.exit(main())
[0.464s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.464s]       abs_idl_file = convert_to_idl(
[0.464s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 29, in convert_to_idl
[0.464s]       return convert_action_to_idl(
[0.464s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/action/__init__.py", line 28, in convert_action_to_idl
[0.464s]       action = parse_action_string(package_name, input_file.stem, content)
[0.464s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 894, in parse_action_string
[0.464s]       raise InvalidActionSpecification(
[0.464s] 
[0.464s]   rosidl_adapter.parser.InvalidActionSpecification: Number of '---'
[0.464s]   separators nonconformant with action definition
[0.464s] 
[0.464s] Call Stack (most recent call first):
[0.464s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.464s]   CMakeLists.txt:16 (rosidl_generate_interfaces)
[0.464s] 
[0.464s] [0m
[0.464s] -- Configuring incomplete, errors occurred!
[0.464s] See also "/home/nicholas/arduinobot_ws/build/arduinobot_msgs/CMakeFiles/CMakeOutput.log".
[0.468s] gmake: *** [Makefile:1395: cmake_check_build_system] Error 1
[0.471s] Invoked command in '/home/nicholas/arduinobot_ws/build/arduinobot_msgs' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/nicholas/arduinobot_ws/install/arduinobot_py_examples:/opt/ros/humble /usr/bin/cmake --build /home/nicholas/arduinobot_ws/build/arduinobot_msgs -- -j16 -l16
